tools/Assembly/antismash/chunking_antismash_with_conditionals/antismash/run_antismash_short.sh

```diff
@@ -1,12 +1,23 @@
 #!/bin/bash
 
-while getopts i:o:g:n:f: option; do
+set -e
+
+while getopts i:o: option; do
 	case "${option}" in
-		i) INPUT=${OPTARG};;
-		o) OUTFOLDER=${OPTARG};;
+	i) INPUT=${OPTARG} ;;
+	o) OUTFOLDER=${OPTARG} ;;
+	*) echo "Invalid option: ${option}" ;;
```

Comment:
I guess we have more bash scripts without *) option...

Reply:
yeah, I'll fix them progressively